integrations/enumeratum/src/test/scala/sttp/tapir/codec/enumeratum/TapirCodecEnumeratumTest.scala

```diff
@@ -180,4 +185,37 @@ object TapirCodecEnumeratumTest {
 
     override def values: scala.collection.immutable.IndexedSeq[TestCharEnumEntry] = findValues
   }
+
+  @default(TestEnumEntryWithNoEncodedDefault.Value2)
+  sealed trait TestEnumEntryWithNoEncodedDefault extends EnumEntry
+
+  object TestEnumEntryWithNoEncodedDefault extends Enum[TestEnumEntryWithNoEncodedDefault] {
+    case object Value1 extends TestEnumEntryWithNoEncodedDefault
+    case object Value2 extends TestEnumEntryWithNoEncodedDefault
+    case object Value3 extends TestEnumEntryWithNoEncodedDefault
+
+    override def values: scala.collection.immutable.IndexedSeq[TestEnumEntryWithNoEncodedDefault] = findValues
+  }
+
+  @default(TestEnumEntryWithSomeEncodedDefault.Value2, encoded = Some(TestEnumEntryWithSomeEncodedDefault.Value2))
+  sealed trait TestEnumEntryWithSomeEncodedDefault extends EnumEntry
+
+  object TestEnumEntryWithSomeEncodedDefault extends Enum[TestEnumEntryWithSomeEncodedDefault] {
+    case object Value1 extends TestEnumEntryWithSomeEncodedDefault
+    case object Value2 extends TestEnumEntryWithSomeEncodedDefault
+    case object Value3 extends TestEnumEntryWithSomeEncodedDefault
+
+    override def values: scala.collection.immutable.IndexedSeq[TestEnumEntryWithSomeEncodedDefault] = findValues
+  }
+
+  @default(TestEnumEntryWithExplicitNoEncodedDefault.Value2, encoded = None)
+  sealed trait TestEnumEntryWithExplicitNoEncodedDefault extends EnumEntry
```

Comment:
The difference is that `TestEnumEntryWithNoEncodedDefault` uses a default value for `encoded` and `TestEnumEntryWithExplicitNoEncodedDefault` sets this value explicitly. Otherwise they are the same, so if you feel that it's a duplication, than I'll remove one of them.

Reply:
it's just the way default parameters work, I don't think we need to test that part of Scala :)

Reply:
OK, I've removed that test.